README.md

```diff
@@ -231,7 +231,7 @@ This is required for OAuth solutions that span more than one sub-domain.
 
 1. Create your OAuth configuration and Fresh plugin.
 
-   ```ts
+   ```ts, ignore
```

Comment:
Because the snippet uses the `$fresh/serve.ts` import specifier is being used. Note: Fresh isn't available on JSR yet.